### PR TITLE
add voellmy friction tpye

### PR DESCRIPTION
**Pull request by awirb**

### CODE REVIEW

File: avaframe/com1DFA/DFAfunctionsCython.pyx [hunk 1]
```diff
@@ -293,6 +294,8 @@ def computeForceC(cfg, particles, fields, dem, dT, int frictType):
           elif frictType == 2:
             # coulomb friction type (bottom shear stress)
             tau = mu * sigmaB
+          elif frictType == 3:
```
Comment (matthiasto) [inline avaframe/com1DFA/DFAfunctionsCython.pyx:297]: maybe add a one line comment

**Review by matthiasto — CHANGES_REQUESTED** [review]

File: avaframe/com1DFA/DFAfunctionsCython.pyx (hunk 1 above)
Comment (awirb) [reply avaframe/com1DFA/DFAfunctionsCython.pyx:297]: added a comment

File: avaframe/com1DFA/com1DFA.py [hunk 2]
```diff
@@ -968,7 +968,7 @@ def DFAIterate(cfg, particles, fields, dem):
     resTypesLast = list(set(resTypes + resTypesReport))
     # derive friction type
     # turn friction model into integer
-    frictModelsList = ['samosAT', 'Coulomb']
+    frictModelsList = ['samosAT', 'Coulomb', 'Voellmy']
     frictType = frictModelsList.index(cfgGen['frictModel']) + 1
```
Comment (fso42) [inline avaframe/com1DFA/com1DFA.py:972]: This fails easily i.e. if someone writes 'coulomb' or ' Coulomb'. I suggest stripping spaces and compare case - insensitive

File: avaframe/com1DFA/com1DFACfg.ini [hunk 3]
```diff
@@ -130,14 +130,18 @@ depMin = 1.0e-6
...
+#+++++++++++++General Friction parameters
+# tan of bed friction angle used for: samosAT, Coulomb, Voellmy
 mu = 0.15500
+#+++++++++++++SamosAt friction model
 Rs0 = 0.222
 kappa = 0.43
 R = 0.05
 B = 4.13
+#+++++++++++++Voellmy friction model
+xsi = 4000.
```
Comment (fso42) [inline avaframe/com1DFA/com1DFACfg.ini:144]: General question: should we start adding units in the description?

**Review by fso42 — CHANGES_REQUESTED** [review]

File: avaframe/com1DFA/com1DFA.py (hunk 2 above)
Comment (awirb) [reply avaframe/com1DFA/com1DFA.py:972]: Done

File: avaframe/com1DFA/com1DFACfg.ini (hunk 3 above)
Comment (awirb) [reply avaframe/com1DFA/com1DFACfg.ini:144]: I made an issue for this

**Pull request merged**